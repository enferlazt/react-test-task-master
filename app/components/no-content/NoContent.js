import React from 'react';

const NoContent = () => (
    <section className="no-content">
        <span className="no-content__picture"></span>
        <div className="no-content__text">
            Enter some ticker symbol here to start
        </div>
    </section>
);

export default NoContent;
