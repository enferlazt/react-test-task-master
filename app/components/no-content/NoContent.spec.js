import { shallow } from 'enzyme';
import React from 'react';
import NoContent from './NoContent';
import '../../../setupTests';

describe("NoContent component", () => {
    it("render component", () => {
        const component = shallow(<NoContent/>);
        expect(component).toMatchSnapshot();
    });
});