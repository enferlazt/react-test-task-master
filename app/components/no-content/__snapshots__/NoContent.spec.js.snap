// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContent component render component 1`] = `ShallowWrapper {}`;
